Sources/ComposableArchitecture/SharedState/Shared.swift

```diff
@@ -12,7 +12,7 @@ import IssueReporting
 /// wrapper.
 @dynamicMemberLookup
 @propertyWrapper
-public struct Shared<Value> {
+public struct Shared<Value: Sendable> {
   private let reference: any Reference
   private let keyPath: AnyKeyPath
```

Comment:
Technically this should now be:

```swift
public struct Shared<Value: Sendable> {
```

…and we should drop the conditional conformance below. But there are some weird interactions with `_AnyKeyPath`. Let's chat about this today.